index.js

```diff
@@ -123,6 +123,7 @@ io.on('connection',(socket) => {
 
 });
 
-http.listen(7777, () => {
-  console.log('listening on *:7777');
+const port = process.argv[2] || 7777;
+http.listen(port, () => {
+  console.log('listening on *:' + port);
```

Comment:
Get the port number to listen to from the second command line argument (node index.js port), instead of hardcoding to 7777. If no port is provided, it falls back to 7777.